### PR TITLE
Convert timezone

### PR DESCRIPTION
**Pull request by bjoernhaeuser**
It looks like changing the session timezone does not change the timestamp output, therefore converting the timezone directly to UTC.

### CODE REVIEW

**Comment by erezsh** [comment]
@bjoernhaeuser Any chance you could include a test for this behavior?

**Comment by bjoernhaeuser** [comment]
@erezsh I was able to add a test-case which only works for snowflake right now. Do you have an idea how to make this run on all databases?

File: tests/common.py [hunk 1]
```diff
@@ -83,7 +85,8 @@ def get_conn(cls: type, shared: bool = True) -> Database:
             _database_instances[cls] = get_conn(cls, shared=False)
         return _database_instances[cls]
 
-    return connect(CONN_STRINGS[cls], N_THREADS)
+    con = sqeleton.connect.load_mixins(AbstractMixin_NormalizeValue)
```
Comment (bjoernhaeuser) [inline tests/common.py:88]: This was required to get the "NormalizeAsString" working, not sure if this is actually required?

File: tests/test_database.py [hunk 2]
```diff
@@ -80,3 +79,33 @@ def test_current_timestamp(self):
         db = get_conn(self.db_cls)
         res = db.query(current_timestamp(), datetime)
         assert isinstance(res, datetime), (res, type(res))
+
+    def test_correct_timezone(self):
+        name = "tbl_" + random_table_suffix()
+        db = get_conn(self.db_cls)
+        tbl = table(db.parse_table_name(name), schema={
+            "id": int, "created_at": "timestamp_tz(9)", "updated_at": "timestamp_tz(9)"
```
Comment (bjoernhaeuser) [inline tests/test_database.py:87]: `timestamp_tz(9)` is database vendor agnostic, unfortunately. I have no idea how to make this work with the generic approach. Any hints on this? Thanks.

File: tests/test_database.py [hunk 3]
```diff
@@ -80,3 +79,33 @@ def test_current_timestamp(self):
...
+
+        db.query(tbl.create())
+
+        tz = pytz.timezone('Europe/Berlin')
+
+        now = datetime.now(tz)
+        db.query(table(db.parse_table_name(name)).insert_row("1", now, now))
+        db.query(db.dialect.set_timezone_to_utc())
+
+        t = db.table(name).query_schema()
```
Comment (bjoernhaeuser) [inline tests/test_database.py:98]: This is refreshing the schema, to actually get the correct timezones/datatype values.

**Comment by bjoernhaeuser** [comment]
@erezsh Would you be able to take another look at this? Thanks.

**Comment by erezsh** [comment]
I will soon.

**Comment by erezsh** [comment]
Thank you for contributing!

**Pull request merged**